### ISSUE TITLE
Doesn't build on Xcode 16.1

### ISSUE THREAD

**Issue by tatsuz0u** [Lumisilk/SwiftUI-AnySubviews#1]
## Description
The latest `main` branch doesn't build on `Xcode 16.1 (16B40)`.

## Error logs

```
.../SwiftUI-AnySubviews-main/Sources/AnySubviews/AnySubviewsCollection.swift:83:36: Sending 'subview' risks causing data races
Task-isolated 'subview' is captured by a main actor-isolated closure. main actor-isolated uses in closure may race against later nonisolated uses

.../SwiftUI-AnySubviews-main/Sources/AnySubviews/AnySubviewsCollection.swift:92:36: Sending 'child' risks causing data races
Task-isolated 'child' is captured by a main actor-isolated closure. main actor-isolated uses in closure may race against later nonisolated uses
```

**Comment by Lumisilk** [comment]
Thanks for reporting! Just released v1.2.0 which should work.

**Comment by tatsuz0u** [comment]
Thank you for the quick fix and for all the work you’ve put into this library—it’s been a huge help!
